### PR TITLE
Adding Jedis HSTRLEN command request #1404

### PR DESCRIPTION
**Pull request by smadasu**
Hi,
Here is the pull request for HSTRLEN command

### CODE REVIEW

File: src/test/java/redis/clients/jedis/tests/JedisTest.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
 public class JedisTest extends JedisCommandTestBase {
   @Test
   public void useWithoutConnecting() {
-    Jedis jedis = new Jedis("localhost");
     jedis.auth("foobared");
+    Jedis jedis = new Jedis("localhost");
```
Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/JedisTest.java]: I think this is by accident. Could you revert it, or could you explain why this change is necessary?

File: src/test/java/redis/clients/jedis/tests/JedisTest.java [hunk 2]
```diff
@@ -188,4 +188,56 @@ public void testBinaryBitfield() {
     }
   }
 
+  @Test
```
Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/JedisTest.java]: We're placing command tests to other places. I guess you referred above tests but above tests (BitField and BinaryBitField) are also wrong, and so it should be moved, too.

Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/JedisTest.java]: HashesCommandsTest is the proper place of hash commands.

Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/JedisTest.java]: One more, you can just use `jedis` without initialize yourself. Please refer other test methods in HashesCommandsTest.

File: src/test/java/redis/clients/jedis/tests/JedisTest.java (hunk 1 above)
Comment (smadasu) [inline src/test/java/redis/clients/jedis/tests/JedisTest.java]: Yes. Thats right. I will fix it Thanks.

**Comment by smadasu** [comment]
Update parameter names and Test.

**Comment by smadasu** [comment]
Sorry I closed PR by mistake

**Comment by marcosnils** [comment]
@smadasu can you please rebase?

**Comment by smadasu** [comment]
New PR after rebasing

**Comment by smadasu** [comment]
Why is it failing just for openjdk7, I don't understand

**Comment by HeartSaVioR** [comment]
Travis CI has some bugs with OpenJDK. Maybe we want to get rid of checking OpenJDK.

**Comment by HeartSaVioR** [comment]
@smadasu Tests still not moved to proper place. If you want me to move them, please let me know. I'll take care of it.

**Comment by smadasu** [comment]
Sure @HeartSaVioR . Go ahead and clean them up. Thanks for your help.

**Comment by marcosnils** [comment]
@HeartSaVioR @smadasu I've re-run the tests for JDK 7 and we're ok now. Sometimes they fail because because of the timeouts.

**Comment by HeartSaVioR** [comment]
OK then LGTM from me.
@marcosnils Please take a look as well. Thanks!

**Comment by marcosnils** [comment]
LGTM!

**Comment by HeartSaVioR** [comment]
Tests polished via 6a933f1 for master, and 128cb2d for 2.10.

@smadasu Thanks for the contribution!

**Comment by marcosnils** [comment]
@HeartSaVioR LGTM!.
